### ISSUE TITLE
Failed to install plugin 'js' with node 12

### ISSUE THREAD

**Issue by paddydeklee** [getgauge/gauge-js#199]
Similar to #182 

## Goal 
Trying to install gauge-js

## Expected behaviour
gauge-js installs successfully on command:
-> gauge install js

## Actual behaviour
gauge-js fails to install via command "gauge install js" or manually from local file:

## See below:

➜  testing gauge install js
.........
  CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/init.o
....
  CXX(target) Release/obj.target/gpr/deps/grpc/src/core/lib/profiling/basic_timers.o
  CXX(target) Release/obj.target/gpr/deps/grpc/src/core/lib/profiling/stap_timers.o
  LIBTOOL-STATIC Release/libgpr.a
  CXX(target) Release/obj.target/grpc_node/ext/byte_buffer.o
In file included from ../ext/byte_buffer.cc:21:
In file included from ../../nan/nan.h:223:
In file included from ../../nan/nan_new.h:189:
../../nan/nan_implementation_12_inl.h:356:37: error: too few arguments to function call, expected 2, have 1
  return v8::StringObject::New(value).As<v8::StringObject>();
         ~~~~~~~~~~~~~~~~~~~~~      ^
/Users/patrickdeklee/.node-gyp/12.1.0/include/node/v8.h:5380:3: note: 'New' declared here
  static Local<Value> New(Isolate* isolate, Local<String> value);
  ^
In file included from ../ext/byte_buffer.cc:21:
In file included from ../../nan/nan.h:223:
In file included from ../../nan/nan_new.h:189:
../../nan/nan_implementation_12_inl.h:356:58: error: expected '(' for function-style cast or type construction
  return v8::StringObject::New(value).As<v8::StringObject>();
                                         ~~~~~~~~~~~~~~~~^
../../nan/nan_implementation_12_inl.h:356:60: error: expected expression
  return v8::StringObject::New(value).As<v8::StringObject>();
                                                           ^
3 errors generated.
make: *** [Release/obj.target/grpc_node/ext/byte_buffer.o] Error 1
Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --library=static_library --module=/private/var/folders/3n/_y9_t62s4v3ds1ggt45v_4180000gn/T/gauge_temp1556637007488503000/node_modules/grpc/src/node/extension_binary/node-v72-darwin-x64-unknown/grpc_node.node --module_name=grpc_node --module_path=/private/var/folders/3n/_y9_t62s4v3ds1ggt45v_4180000gn/T/gauge_temp1556637007488503000/node_modules/grpc/src/node/extension_binary/node-v72-darwin-x64-unknown --napi_version=4 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v72' (1)
Failed to install plugin 'js'.
Reason: exit status 1

➜  testing gauge version 
Gauge version: 1.0.5
Commit Hash: 562f036

Plugins
-------
html-report (4.0.8)
screenshot (0.0.1)

**Comment by zabil** [comment]
Can you share the node js version and OS version?

**Comment by paddydeklee** [comment]
➜  node -v
v12.1.0

➜  MacOS 
Mojave 
v10.14.2

**Comment by zabil** [comment]
Thanks for reporting this. 
Taiko does not work on node 12 at the moment. Will take a look at this soon.
Meanwhile please try it with node versions 8 and 10.

**Comment by nehashri** [comment]
This seems to be an issue with gauge-js and not Taiko. It doesn't seem to install with node 12.

**Comment by Apoorva-GA** [comment]
Fix should be available in gauge-js nightly >= 3/05/19

Note:Need to check backward compatibility with node 8, 10 and 11.

**Comment by Debashis9012** [comment]
``` gauge install js``` working fine with node version 12. 
I have checked backward compatibility with the node version 8,10 and 11 able to install js plugin successfully.
Tested with the fix version ``` gauge-js-2.3.5.nightly-2019-05-03 ```

**Comment by jakobheine** [comment]
I'm still getting this error while manuallty trying to install the js plugin with `gauge install js`

**Environment information:**
Gauge version: 1.0.6
Node version: 13.0.1
OS: linux, ubuntu 19.10, eoan ermine
IDE: Visual Studio Code

I wanted to install it manually because I got this error message while trying to run the spec.

> Current gauge language runner is not compatible with gauge LSP. Some of the editing feature will not work as expected
